src/conditions/Condition.js

```diff
@@ -0,0 +1,60 @@
+const Joi = require('joi')
+
+class Condition {
+
+  static get SCHEMA() {
+    return Joi.object({
+      operator: Joi.string().allow(null).default(null),
```

Comment:
Might be worth already defining our operators here, from what I've seen they are also usually capitalized by convention